src/functionalTest/kotlin/com/autonomousapps/fixtures/DataBindingWithExpressionsProject.kt

```diff
@@ -0,0 +1,48 @@
+package com.autonomousapps.fixtures
+
+import com.autonomousapps.advice.Advice
+
+class DataBindingWithExpressionsProject(
```

Comment:
Please rewrite this fixture to use a similar style to `AttrResWithNullProject` from the spec just above your new one. This is the first style of fixture used in this project and... I hate it. I'd like to eliminate it entirely at some point.

Reply:
Sure, not a problem. Please check my latest commit with an updated `DataBindingWithExpressionsProject` implementation.